ocp_utilities/vault_utils.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import requests
+import urllib3
+from hvac import Client as hvacClient
+from urllib3.exceptions import InsecureRequestWarning
+
+urllib3.disable_warnings(InsecureRequestWarning)
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_vault_config(vault_url, data_path, role_id, secret_id, verify=True):
+    """
+    Fetch vault data config
+    Args:
+        vault_url (str): vault url host to create client
+        data_path (str): path within vault to read from 'apps/data/..
+        role_id (str): for example: 'mps-qe'
```

Comment:
please update to explain what this is (based on vault) and remove any internal examples. this repo is public